libraries/wallet/include/graphene/wallet/wallet.hpp

```diff
@@ -476,6 +476,13 @@ class wallet_api
        */
       account_object                    get_account(string account_name_or_id) const;
 
+      /** Returns string representation of an account ID.
+       *
+       * @param id the ID of the account to be converted to string
+       * @returns string representation of account ID
+       */
+      string                    account_id_to_string(account_id_type id) const;
```

Comment:
Since we already have a function in the impl class, I think we don't need it here.

Reply:
silly me, fixed here https://github.com/bitshares/bitshares-core/pull/1323/commits/cab7ff4dae14d713bc21698f57f3ca41c37d585d thanks.